### PR TITLE
[TS-198] fix fields for validate

### PR DESCRIPTION
**Pull request by jenchik**

### CODE REVIEW

**Review by pcherednichenko — DISMISSED** [review]

File: helpers/collection/form.go [hunk 1]
```diff
@@ -85,17 +86,26 @@ func (f *Form) ApplyPager(opts *Options) (err error) {
 		val   int64
 	)
 
-	if val, err = f.Limit.Int64(); err != nil {
-		return err
-	} else if val > 0 {
-		limit = int(val)
+	if len(f.Limit) > 0 {
+		if val, err = f.Limit.Int64(); err != nil {
+			return err
```
Comment (im-kulikov) [inline helpers/collection/form.go]: `%s, must be a number`, "limit"

and make it const

File: helpers/collection/form.go [hunk 2]
```diff
@@ -85,17 +86,26 @@ func (f *Form) ApplyPager(opts *Options) (err error) {
...
+		} else if val <= 0 {
+			return errors.New("'limit' must be greater than '0'")
```
Comment (im-kulikov) [inline helpers/collection/form.go]: `"'%s' must be greater than '0'", "limit"`

and make it const

Comment (im-kulikov) [inline helpers/collection/form.go]: make it as error var

File: helpers/collection/form.go [hunk 3]
```diff
@@ -85,17 +86,26 @@ func (f *Form) ApplyPager(opts *Options) (err error) {
...
 	}
 	opts.Query.Limit(limit)
 
-	if val, err = f.Offset.Int64(); err != nil {
-		return err
+	if len(f.Offset) > 0 {
+		if val, err = f.Offset.Int64(); err != nil {
+			return err
+		} else if val < 0 {
+			return errors.New("'offset' must be greater or equal than '0'")
```
Comment (im-kulikov) [inline helpers/collection/form.go]: make it as error var

**Review by im-kulikov — APPROVED** [review]
👍🏻

**Pull request merged**